src/components/molecules/MovableBoundary/index.jsx

```diff
@@ -0,0 +1,166 @@
+//#region > Imports
+//> React
+// Contains all the functionality necessary to define React components
+import React, { lazy, Suspense } from "react";
+// DOM bindings for React Router
+import { Link, withRouter } from "react-router-dom";
+// React PropTypes
+import PropTypes from "prop-types";
+
+//> MDB Sortable
+// MDB plugin for sortable item lists
+import MDBSortable from "mdb-react-sortable";
+
+//> SCSS
```

Comment:
All the other files used to have CSS and not SCSS, is this a new standard or a mistake?
```suggestion
//> CSS
```